data/json/materials.json

```diff
@@ -380,7 +380,7 @@
         ],
         "acid_resist" : 3,
         "elec_resist" : 2,
-        "fire_resist" : 2,
+        "fire_resist" : 3,
```

Comment:
Pretty confident that stone structures can withstand fire (it's what fireplaces are made of, after all) so I'm thinking 4.  (Melts in lava--by definition, hot enough to melt rock--is another story.)